src/styles/_animation.scss

```diff
@@ -0,0 +1,35 @@
+@keyframes min-handle-animation {
+  from {
+    transform: rotate(0deg);
+  }
+  50% {
```

Comment:
what is this breakpoint for?

Reply:
this breakpoint for changing **background-color** to **seagreen** when minute hand show on 30 min